src/modules/feature-modules/innovator/pages/innovation/manage/manage-stop-sharing-overview.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { CoreComponent } from '@app/base';
+import { TableModel } from '@app/base/models';
+import { DateISOType } from '@app/base/types';
+import { InnovationsService } from '@modules/shared/services/innovations.service';
+import { InnovationStatusEnum } from '@modules/stores/innovation';
+
+
+
+@Component({
+  selector: 'app-innovator-pages-innovation-manage-stop-sharing-overview',
+  templateUrl: './manage-stop-sharing-overview.component.html'
+})
+export class PageInnovationManageStopSharingOverviewComponent extends CoreComponent implements OnInit {
+
+  innovationId: string;
+
+  innovations: TableModel<{ id: string, name: string, statusUpdatedAt: null | DateISOType }>;
+
+  constructor(private activatedRoute: ActivatedRoute, private innovationsService: InnovationsService,) {
+
+    super();
+
+    this.innovationId = this.activatedRoute.snapshot.params.innovationId;
+
+    this.setPageTitle('Stop sharing this innovation');
+    this.setBackLink('Go back', `/innovator/innovations/${this.innovationId}/manage-innovation`);
+
+     this.innovations = new TableModel({
+      visibleColumns: {
+        innovations: { label: 'Innovations' },
+        statusUpdatedAt: { label: 'Date' },
+        actions: { label: '', align: 'right' }
+      }
+    });
+
+  }
+
+  ngOnInit() {
+
+     this.innovationsService.getInnovationsList({ queryParams: { take: 100, skip: 0, filters: { status: [InnovationStatusEnum.PAUSED] } } }).subscribe((innovations) => {
```

Comment:
Same list behaviour, this should be for a single innovation I believe

Reply:
In this case, a table with all of the innovations that have been stopped is shown to the user. I don't know if this table should be removed. If this was removed, then it is only necessary to inform the user on the "Manage innovation - Stop sharing this innovation" section that he already has stopped sharing the innovation

![image](https://user-images.githubusercontent.com/124596230/222125344-66cd7ad3-33a7-4294-a88b-7fde89cb0875.png)